src/vtk_calculator.py

```diff
@@ -11,193 +11,189 @@
 
 """Evaluates a function on a given mesh, using the VTK Calculator."""
 
-"""
-This calculator can calculate vector or scalar field on given mesh.
 
-Example usage
-
-Scalar calculation and writing to given file
-
-
-vtk_calculator.py -m inputmesh.vtk -f "exp(cos(x)+sin(y))" --data "e^(cos(x)+sin(y))" -o outputmesh.vtk
-
-
-Vector field calculation and appends to input mesh
-
-
-vtk_calculator.py -m "inputmesh.vtk" -f "x*iHat+cos(y)*jHat-sin(z)*kHat" -d "MyVectorField"
-There is also a diff mode which provides statistic between input data and function calculated
-(Note that it only works for scalar data)
```

Comment:
Why removing the documentation here?

Reply:
Do we really need documentation here? If these tools will be used as an installed binary, let's keep them clean and move documentation to the website.